### PR TITLE
Implemented pagination for upcoming and past sessions

### PR DESCRIPTION
**Pull request by BenjaminSchaaf**

### CODE REVIEW

File: coviu-calls.php [hunk 1]
```diff
@@ -777,6 +765,77 @@ function cvu_session_delete( $post, $options ) {
 	}
 }
 
+function cvu_get_active_sessions($coviu, $params = array()) {
+	// Sessions are a maximum of 12 hours long
+	// So we can grab sessions within a 12 hour radius to get all currently active ones
+	$now = new DateTime();
```
Comment (silviapfeiffer) [inline coviu-calls.php]: I think you need to pass $now to the three functions, so you don't re-calculate the DateTime() every time - they might be slightly off

File: coviu-calls.php [hunk 2]
```diff
@@ -435,66 +435,32 @@ function delete_host(host_id) {
...
-			}
-		}
-		array_values($sessions);
-
-		// determine split point in remaining sessions
-		foreach ($sessions as $key => $session) {
-			if ($now > $session['start_time']) break;
-			$upcoming_split_index++;
+		$params = array(
+			'page_size' => 10,
```
Comment (silviapfeiffer) [inline coviu-calls.php:441]: move it into the past sessions retrieval function - I think we should probably retrieve all current and all future sessions

**Review by silviapfeiffer — COMMENTED** [review]
Nice - just some small fixes.

File: coviu-calls.php (hunk 2 above)
Comment (BenjaminSchaaf) [reply coviu-calls.php:441]: Aren't a lot of appointments booked months in advance? Couldn't making future sessions non-paged become an issue pretty quickly?

Comment (silviapfeiffer) [reply coviu-calls.php:441]: maybe - ok, leave it

**Pull request merged**